specs/deneb/p2p-interface.md

```diff
@@ -188,16 +217,23 @@ Request Content:
 ```
 
 Response Content:
+
+```python
+class BlobSidecars(Container):
```

Comment:
what about `MultiBlobSidecar`?

Reply:
not bad, not bad - I was thinking `BlockBlobSidecars` to highlight that they belong to the same block, but ... I'll let this stew until we're ready to merge :)